tripy/tests/integration/test_pooling.py

```diff
@@ -31,13 +31,20 @@ class TestPooling:
         ],
     )
     @pytest.mark.parametrize("dtype", [tp.float32, tp.float16, tp.int8])
-    def test_maxpool_2d(self, kernel_dims, stride, padding, dtype):
+    @pytest.mark.parametrize("pool_type", ["max", "avg"])
+    def test_maxpool_2d(self, kernel_dims, stride, padding, dtype, pool_type):
         inp_tp = tp.reshape(tp.arange(64, dtype=dtype), (1, 1, 8, 8))
-        out = tp.maxpool(inp_tp, kernel_dims=kernel_dims, stride=stride, padding=padding)
-        out_torch = torch.from_dlpack(out).to("cpu")
-
         torch_padding = (padding[0][0], padding[1][0])
-        pool_torch = torch.nn.MaxPool2d(kernel_size=kernel_dims, stride=stride, padding=torch_padding)
+
+        if pool_type == "max":
+            out = tp.maxpool(inp_tp, kernel_dims=kernel_dims, stride=stride, padding=padding)
+            pool_torch = torch.nn.MaxPool2d(kernel_size=kernel_dims, stride=stride, padding=torch_padding)
+        elif pool_type == "avg":
+            pytest.skip("https://github.com/NVIDIA/TensorRT-Incubator/issues/237: Average pooling is not functional.")
```

Comment:
Can not be merged until https://github.com/NVIDIA/TensorRT-Incubator/issues/237 is fixed.

Reply:
Will take another look once it is fixed.